README.md

```diff
@@ -407,6 +408,24 @@ Currently all subworkflow DAGs must be in examples folder
 
 * [Subworkflow conversion expects to be run in examples](https://github.com/GoogleCloudPlatform/oozie-to-airflow/issues/213)
 
+## EL functions support
+
+Currently there are a few `wf:functions` which are not implemented:
+
+* [`wf:actionTrackerUri`](https://github.com/GoogleCloudPlatform/oozie-to-airflow/issues/372)
+* [`wf:actionExternalId`](https://github.com/GoogleCloudPlatform/oozie-to-airflow/issues/371)
+* [`wf:actionData`](https://github.com/GoogleCloudPlatform/oozie-to-airflow/issues/370)
+* [`wf:run`](https://github.com/GoogleCloudPlatform/oozie-to-airflow/issues/369)
+* [`wf:errorMessage`](https://github.com/GoogleCloudPlatform/oozie-to-airflow/issues/368)
+* [`wf:errorCode`](https://github.com/GoogleCloudPlatform/oozie-to-airflow/issues/367)
+* [`wf:transition`](https://github.com/GoogleCloudPlatform/oozie-to-airflow/issues/366)
+* [`wf:callback`](https://github.com/GoogleCloudPlatform/oozie-to-airflow/issues/365)
+* [`wf:group`](https://github.com/GoogleCloudPlatform/oozie-to-airflow/issues/364)
+* [`wf:appPath`](https://github.com/GoogleCloudPlatform/oozie-to-airflow/issues/363)
+
+Additionally some already implemented functions may not preserve full logic of the original el-expression.
```

Comment:
```suggestion
Additionally some already implemented functions may not preserve full logic of the original EL expression.
```

Reply:
I think we should probably describe why.

Reply:
Agree. Some more explanation might be needed like - "It's difficult to implement it in generic-enough way to cover all possible cases, it's much easier to leave the implementation of those functions to the user. It's perfectly possible to provide your own implementation of each of those functions if you need to customise it and in many cases it will be easier if it's specific implementation rather than generic one"